2kki.js

```diff
@@ -11,13 +11,16 @@ function send2kkiApiRequest(url, callback) {
     const req = new XMLHttpRequest();
     req.responseType = 'json';
     req.open('GET', url);
+    req.timeout = 10000;
     req.send();
 
-    req.onload = _e => {
+    let onReqEnd = _e => {
```

Comment:
I would normally make this const since the value is not changed

Reply:
I agree; it should be `const`. I have the change pushed to my fork, but I don't see a way to reopen this PR. Unsure what to do, since making a new PR only to change `let` to `const` seems silly to me.

Reply:
It is, which is why I merged it as-is. It's not enough to warrant an additional commit.